agent/tool-scripts/pprof

```diff
@@ -146,7 +160,12 @@ ose_pprof() {
 
 collect_data() {
     while true; do
-            $tool_bin tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_.txt http://localhost:6060/debug/pprof/profile
+        if [ $(systemctl list-unit-files | grep atomic-openshift-master.service | awk {'print $2'}) == "masked" ]; then
+            $tool_bin tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_controller.txt http://localhost:6060/debug/pprof/profile &
```

Comment:
it can be 
go tool pprof -text -seconds=5 -output testfile_api.txt http://localhost:6061/debug/pprof/profile & 
or 
go tool pprof -text -seconds=5 -output testfile_api.txt http://localhost:6061/debug/pprof/profile 

on any openshift installation above will exit after 5 seconds

t will just grab profile from http://localhost:6061/debug/pprof/profile and exit itself ( after 5 seconds in above example ) and exist it ends iteself and in pbench context we will search for pprof related processes in ps output and kill all when it comes to stop tools. pprof can live without this too, but to sure there are not some pprof running at time when stop tools is called, we just kill all.  For pprof tool, it does only query against internal pprof web server and save that profile. We decided to collect these information every 10 seconds. However, your proposal to make it better is welcome